### PR TITLE
Fix ordering in README

### PR DESCRIPTION
**Pull request by nikhita**
@amarlearning Let me know if this is any better.

### CODE REVIEW

**Comment by nikhita** [comment]
@amarlearning  And thanks for your suggestion about left-aligning it! It looks waaayyy better! :raised_hands:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ To add your own writeup, please follow these instructions:
 3. Update the README in the following format:
 
     ```
-    | [ProjectName, OrgName](writeups/orgname_projectname_yourname.md) | [YourName](https://github.com/your-github-username) |
+    | OrgName | [ProjectName](writeups/orgname_projectname_yourname.md) | [YourName](https://github.com/your-github-username) |
```
Comment (amarlearning) [inline CONTRIBUTING.md:10]: One should also add the org link as well

**Review by amarlearning — CHANGES_REQUESTED** [review]
LGTM! cosmetic changes :sweat_smile:

**Review by amarlearning — APPROVED** [review]

**Comment by nikhita** [comment]
Btw added the org website as the link (not the github repo) because the website would have other useful links like mailing lists, etc.

**Pull request merged**